README.md

```diff
@@ -52,6 +51,100 @@ as they will make continuous verification of your plugin's functionality much ea
 In order to use these tests, you will need to address the "FIXME" messages left in places where
 plugin-writer intervention is required.
 
+# Generate a template_config.yaml for an existing plugin
+
+The ``plugin-template`` script can update a plugin with the latest Travis CI configuration. Plugin
+authors can select which parts of the CI pipeline are included by providing a template_config.yaml
+in the root of the plugin repository. Run the following command to generate a template_config.yaml
+with all the defaults.
+
+    ``$ ./plugin-template --generate-config plugin_name plugin_app_label``
+
+This is going to generate a template_config.yaml with the following settings:
```

Comment:
If I get the code correctly, `--generate-config` will not generate a new default config when  a config file is already existing (it will parse the existing config and add defaults for new settings that are not part of the file).

I like this behavior, but I think it should be made clearer here. Judging from the docs, I would have expected it to just generate a new config file.

Reply:
I'll add this clarification.